### PR TITLE
Set DNS servers on public network

### PR DESCRIPTION
**Pull request by yussufsh**
Signed-off-by: Yussuf Shaikh <yussuf.shaikh@ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/88#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/release-4.5/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@yussufsh: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pull-ocp4-upi-powervs-terraform-validate | e015f048dced3ef28386461a6b10cbecf21d4c37 | [link](https://prow.k8s-prow-c7f9fbfa2832409e2a3a6666e9dc52a3-0000.us-south.containers.appdomain.cloud/view/s3/prow-logs/pr-logs/pull/ocp-power-automation_ocp4-upi-powervs/88/pull-ocp4-upi-powervs-terraform-validate/1307013996938792960) | `/test pull-ocp4-upi-powervs-terraform-validate`

[Full PR test history](https://prow.k8s-prow-c7f9fbfa2832409e2a3a6666e9dc52a3-0000.us-south.containers.appdomain.cloud/pr-history?org=ocp-power-automation&repo=ocp4-upi-powervs&pr=88). [Your PR dashboard](https://prow.k8s-prow-c7f9fbfa2832409e2a3a6666e9dc52a3-0000.us-south.containers.appdomain.cloud/pr?query=is:pr+state:open+author:yussufsh).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: ocp.tf
Comment (mkumatag) [inline ocp.tf:40]: Where is this dns_forwarders defined?

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: ocp.tf
Comment (yussufsh) [reply ocp.tf:40]: This is coming from variables. Initial use was to add forwarder in named config.

Comment (mkumatag) [reply ocp.tf:40]: 👍

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/88#pullrequestreview-491674553" title="LGTM">mkumatag</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/88#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/release-4.5/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @yussufsh

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/88#pullrequestreview-491679979" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/88#pullrequestreview-491674553" title="LGTM">mkumatag</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/88#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/release-4.5/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by mkumatag** [comment]
I was fixing the job to test it :P

**Pull request merged**
